src/components/Legend/PumpsColorLegend.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  workingColor,
+  defaultColor,
+  brokenColor,
+  lockedColor,
+} from '../map/colors';
+import { PumpsDot } from './PumpsDot';
+import { PumpLabel } from './PumpLabel';
+import { UnstyledFlex } from './UnstyledFlex';
+import { FlexRow } from './FlexRow';
+
+export const PumpsColorLegend: React.FC = () => {
+  return (
+    <UnstyledFlex>
+      <FlexRow>
+        <PumpsDot color={workingColor.hex} size={13} />
+        <PumpLabel>{'funktionsfähig'}</PumpLabel>
```

Comment:
No need to wrap the string into curly braces and apostrophes. Write `<PumpLabel>funktionsfähig</PumpLabel>` instead